src/app/Layout/Presentation/Views/DistrictOverview.tsx

```diff
@@ -1,50 +1,105 @@
 import ReactTable from "react-table";
 import { DistrictResult } from "../../Interfaces/Results";
+import { toSum, toMean, toMax, toMin } from "../Utilities/ReduceUtilities";
 
 export interface DistrictOverviewProps {
     districtResults: DistrictResult[];
+    districtWidth: number;
 }
 
 export class DistrictOverview extends React.Component<DistrictOverviewProps, {}> {
     render() {
+        const data = this.props.districtResults;
+        const highestVotingPower = data.map((value) => value.votesPerSeat).reduce(toMin);
+        const lowestVotingPower = data.map((value) => value.votesPerSeat).reduce(toMax);
+        const averageVotingPower = data.map((value) => value.votesPerSeat).reduce(toMean);
         return (
-            <ReactTable
-                className="-highlight -striped"
-                defaultPageSize={19}
-                showPaginationBottom={false}
-                data={this.props.districtResults}
-                columns={[
-                    {
-                        Header: "Fylke",
-                        accessor: "name"
-                    },
-                    {
-                        Header: "Stemmer",
-                        accessor: "votes"
-                    },
-                    {
-                        Header: "Prosent",
-                        accessor: "percentVotes"
-                    },
-                    {
-                        Header: "Distrikt",
-                        accessor: "districtSeats"
-                    },
-                    {
-                        Header: "Utjevning",
-                        accessor: "levelingSeats"
-                    },
-                    {
-                        Header: "Sum",
-                        accessor: "totalSeats"
-                    },
-                    {
-                        Header: "Stemmer/mandat",
-                        accessor: "votesPerSeat"
-                    }
-                ]}
-            />
+            <React.Fragment>
+                <h2>Fylkesoversikt</h2>
+                <span>
+                    {"En stemme i "}
+                    <strong>{data.find((entry) => entry.votesPerSeat === highestVotingPower)!.name}</strong>
+                    {" hadde mest innflytelse, og telte "}
+                    {((1 / highestVotingPower / (1 / averageVotingPower)) * 100).toFixed(2) + "%"}
+                    {" av en gjennomsnittlig stemme"}
+                </span>
+                <span>
+                    {", mens en stemme i "}
+                    <strong>{data.find((entry) => entry.votesPerSeat === lowestVotingPower)!.name}</strong>
+                    {" hadde minst innflytelse, og bare telte "}
+                    {((1 / lowestVotingPower / (1 / averageVotingPower)) * 100).toFixed(2) + "%!"}
+                    {" En stemme i det mest innflytelsesrike fylket telte altså "}
+                    {((1 / highestVotingPower / (1 / lowestVotingPower)) * 100).toFixed(2) + "%"}
```

Comment:
The results seem a bit strange, I may be misinterpreting the numbers, but it doesn't feel intuitive that 212.52% influence is 256.23% more influential than 82.94%. Surely it would be 212.52 - 82.94 % more influential?

Reply:
You are definitely misinterpreting the numbers/maths going on here. This is all relative to the average. It might be more clear in the next push I make following the refactoring that moves the maths into variable names.

It's also debatable whether this is the best way of presenting the statistics -- some people prefer the "more than" approach rather than the "of something" approach.